api/models/OfficerManager.js

```diff
@@ -0,0 +1,52 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+const OfficersSchema = new Schema(
```

Comment:
why not call it `OfficerSchema`? The naming convention seems to be singular
![image](https://user-images.githubusercontent.com/36345325/69918658-c33d2380-1429-11ea-907e-2bf2fd88c81d.png)